config/opal_config_hwloc.m4

```diff
@@ -106,7 +106,9 @@ AC_DEFUN([OPAL_CONFIG_HWLOC], [
     OPAL_3RDPARTY_EXTRA_DIST="$OPAL_3RDPARTY_EXTRA_DIST hwloc_tarball"
     OPAL_3RDPARTY_DISTCLEAN_DIRS="$OPAL_3RDPARTY_DISTCLEAN_DIRS hwloc_directory"
 
+    AC_SUBST(opal_hwloc_CPPFLAGS)
```

Comment:
It doesn't really matter, but we don't need to AC_SUBST opal_hwloc_CPPFLAGS, because they are added top the top-level CPPFLAGS.  I've actually debated ditching the explicit opal_hwloc_LIBS and opal_hwloc_LDFLAGS in the Makefile.am, which is necessary because of the 3rd-party internal build, and using something similar to the PMIX_FINAL_LIBS trick pmix uses (so that LIBS isn't updated until after all configure tests run), but for reasons I can't remember, dropped that.